src/nvim/eval.h

```diff
@@ -3,6 +3,47 @@
 
 #include "nvim/profile.h"
 
+// All user-defined functions are found in this hashtable.
+hashtab_T func_hashtab;
```

Comment:
Should be a declaration, like `extern hashtab_T func_hashtab;` and only be defined in one file (eval.c?).